web/html/src/manager/recurring/recurring-actions-details.tsx

```diff
@@ -272,6 +297,7 @@ class RecurringActionsDetails extends React.Component<RecurringActionsDetailsPro
             </Table>
           </div>
         )}
+        {!(this.props.data.actionType === "PLAYBOOK" && this.state.details) ? null : this.showPlaybookDetails()}
```

Comment:
Maybe we could lift values like `"PLAYBOOK"` out into an enum or something similar?